pages/index.js

```diff
@@ -0,0 +1,56 @@
+import Head from 'next/head';
+import Link from 'next/link';
+import s3 from '../lib/aws';
+
+function Home({ data }) {
+  return (
+    <div className='container mx-auto'>
+      <Head>
+        <title>Scott King Photography</title>
+        <meta name="description" content="Photography works by Scott King" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <main>        
+            {/* {data.map((e, i) => {
+              return (
+                <div key={`folder-${i}`} className={`block transform transition-colors duration-200 hover:text-neutral-900 dark:hover:text-white text-base capitalize cursor-pointer`}>
+                  <Link href={{
+                    pathname: `/photos/${e.path}`,
+                  }}>
+                    <h2 className='capitalize'>{e.name}</h2>
+                  </Link>
+                </div>    
+              )
+            })} */}
```

Comment:
Keeping this as a placeholder for now. I think there may be a way to do this without too much hassle.